Entities/AHuman.cpp

```diff
@@ -4099,65 +4127,63 @@ void AHuman::Update()
         m_pBGLeg->SetTargetPosition(m_pBGFootGroup->GetLimbPos(m_HFlipped));
     }
 
-	if (m_pFGArm && m_pFGArm->IsAttached()) {
+	if (m_pFGArm) {
 		float affectingBodyAngle = 0.0F;
 		if (m_FGArmFlailScalar != 0 && m_SharpAimDelay != 0) {
 			float aimScalar = std::min(static_cast<float>(m_SharpAimTimer.GetElapsedSimTimeMS()) / static_cast<float>(m_SharpAimDelay), 1.0F);
 			float revertScalar = std::min(static_cast<float>(m_SharpAimRevertTimer.GetElapsedSimTimeMS()) / static_cast<float>(m_SharpAimDelay), 1.0F);
 			aimScalar = (aimScalar > revertScalar) ? aimScalar : 1.0F - revertScalar;
 
-			affectingBodyAngle = std::abs(std::sin(m_Rotation.GetRadAngle())) * m_Rotation.GetRadAngle() * m_FGArmFlailScalar * (1.0F - aimScalar);
+			affectingBodyAngle = std::abs(std::sin(rot)) * rot * m_FGArmFlailScalar * (1.0F - aimScalar);
 		}
 		m_pFGArm->SetRotAngle(affectingBodyAngle + m_AimAngle * static_cast<float>(GetFlipFactor()));
 
         if (m_Status == STABLE) {
             if (m_ArmClimbing[FGROUND]) {
-                // Can't climb with anything in the arm?
-                //UnequipBGArm();
                 m_pFGArm->ReachToward(m_pFGHandGroup->GetLimbPos(m_HFlipped));
             } else if (!m_pFGArm->IsReaching()) {
-                // This will likely make the arm idle since the target will be out of range
-                m_pFGArm->Reach(m_pFGHandGroup->GetLimbPos(m_HFlipped));
+				// Use an unreachable position to force this arm to idle, so it wont bug out where the AtomGroup was left off
```

Comment:
I guess using Vector here is 99% safe but theoretically unsafe if someone makes a ladder to the top left corner of the screen lol. Could use Pos + arm length + a bunch or something I guess. Nbd keeping it as `Vector()` though

Reply:
And then what? The player can barely see the arm bug out at that height! :P I assure you it's fine, I've seen crazier things happen at `0, 0` lol

Reply:
Haha fair enough, if someone makes it up there they deserve to have some weirdness happen :P
Feel free to leave this be and mark it resolved if you'd like.